data/json/mapgen/microlab/microlab_special_tiles.json

```diff
@@ -33,7 +33,7 @@
       ],
       "palettes": [ "microlab" ],
       "terrain": { "G": "t_card_science" },
-      "items": { "f": { "item": "mutagens", "chance": 60, "repeat": [ 1, 4 ] } },
```

Comment:
This room is pretty rare all in all, it probably spawns at a rate of 1 ever 40ish overmap tiles, and more importantly is explicitly shaped as a mutagen development/testing room. 

It can be reasonable for the mutagen kit spawns to maintain their original frequency, or to just slightly reduce the repeat chances.